dimagi/data/toolkits/commcare_evidence_base.py

```diff
@@ -33,7 +36,7 @@
 An RCT in South Africa found that
 FLWs using CommCare for
 cardiovascular disease (CVD) screenings
-took 75% less time...
+took 75% less time period.
```

Comment:
remove the word "period". just end the sentence with a period "."

Reply:
Updated .